### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by emailboxu**
~3851

### CODE REVIEW

File: ITEM_en.tsv [hunk 1]
```diff
@@ -3798,57 +3798,57 @@ ITEM_20150323_004071	Formine Armor parts. It is used to pad one's back.
...
-ITEM_20150323_004076	Bag of Teopilis
+ITEM_20150323_004075	A rare, cold body fluid that can be found from monsters. Use it to cool the altar.
+ITEM_20150323_004076	Teopilis's Bag
 ITEM_20150323_004077	Picture Fragment
-ITEM_20150323_004078	Empty frame
-ITEM_20150323_004079	Pemidian Outskirt Warp Scroll
-ITEM_20150323_004081	Strong string
-ITEM_20150323_004082	A strong string obtained from monster. It is useful to tie something.
+ITEM_20150323_004078	Empty Frame
+ITEM_20150323_004079	Femidian Outskirts Warp Scroll
```
Comment (Sunaries) [inline ITEM_en.tsv]: Femidian -> Fedimian

File: ITEM_en.tsv [hunk 2]
```diff
@@ -3798,57 +3798,57 @@ ITEM_20150323_004071	Formine Armor parts. It is used to pad one's back.
...
-ITEM_20150323_004130	Blunt tree to be used to open the tree liquor barrel. The durability is bad as simply made.
-ITEM_20150323_004135	Bitter drink
-ITEM_20150323_004142	Full Poison pot
-ITEM_20150323_004143	A pot full of a monster's poison.
+ITEM_20150323_004089	A nutritious tonic made by diluting the concentrate that the prince gave you with Scorpio Venom.
+ITEM_20150323_004096	Well-Grown White Radish
+ITEM_20150323_004097	A fully grown white radish freshly picked from the fields. It looks healthy, without blemish.
+ITEM_20150323_004098	Roughly Chopped Wood
+ITEM_20150323_004102	First Empty bottle
+ITEM_20150323_004104	A bottle filled with monster spirits
```
Comment (Sunaries) [inline ITEM_en.tsv]: A bottle filled with monster spirits -> A bottle filled with a monster's energy. **or** A bottle filled with the energy of a monster.

File: ITEM_en.tsv [hunk 3]
```diff
@@ -3798,57 +3798,57 @@ ITEM_20150323_004071	Formine Armor parts. It is used to pad one's back.
...
+ITEM_20150323_004105	A bottle filled with monster spirits.
+ITEM_20150323_004110	Keposid's Evil Soul
```
Comment (Sunaries) [inline ITEM_en.tsv]: Soul -> Aura/Energy

File: ITEM_en.tsv [hunk 4]
```diff
@@ -3798,57 +3798,57 @@ ITEM_20150323_004071	Formine Armor parts. It is used to pad one's back.
...
+ITEM_20150323_004113	A bottle filled with a Keposid's wicked soul
```
Comment (Sunaries) [inline ITEM_en.tsv]: A bottle filled with a Keposid's wicked soul -> A bottle filled with Keposid's evil aura/energy.

File: ITEM_en.tsv [hunk 5]
```diff
@@ -3798,57 +3798,57 @@ ITEM_20150323_004071	Formine Armor parts. It is used to pad one's back.
...
+ITEM_20150323_004167	A list of signatures of people agreeing to revolt.
+ITEM_20150323_004170	Coarse Bark
+ITEM_20150323_004171	A Tipio's Skin that's needed to recruit the Peltasta's assistance.
+ITEM_20150323_004172	Priest Yosana's Research
+ITEM_20150323_004173	A research report written by Priest Yosana. It contains research on the types of curses.
+ITEM_20150323_004174	Priest Roana's Research
+ITEM_20150323_004175	A research report written by Priest Roana. It contains research on how curses are made.
+ITEM_20150323_004176	Priest Prozit's Research
+ITEM_20150323_004177	A research report written by Priest Prozit. It contains research on the effects of curses.
+ITEM_20150323_004178	Relic that is treasured by the Church. Unfortunately it's been discarded across the Cathedral.
```
Comment (Sunaries) [inline ITEM_en.tsv]: Relic -> A relic ...

**Comment by imcgames** [comment]
:+1:

**Pull request merged**